### DISCUSSION TITLE
Solution for Inventory in GODOT

### DISCUSSION THREAD

**Discussion by Elisicc** [community/community#142573] [Programming Help]
### Body

Hello, i want to make a 2D "list" inventory, does someone have ideas how to do it? 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Tristan-BS** [comment]
Hi there, yessir i did it already.
I made VBox  with a simple background and an amount of Labels you want as Slots as childs of the VBox.
Just insert the Name of items into the labels and a VERY SIMPLE Inventory is done. xD

Hope this helps :3

**Reply by Elisicc** [reply]
Oh, thats sounds great for an simple inventory Thank you for the help :)

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142574#discussioncomment-11055482) for reference to why this discussion was closed and locked. Thanks!
